lib/octobox.rb

```diff
@@ -0,0 +1,31 @@
+require 'octobox/configurator'
+
+module Octobox
+  class << self
+    def config
+      @config ||= Configurator.new
+      if block_given?
+        yield @config
+      end
+      @config
+    end
+
+    def refresh_interval_enabled?
+      config.minimum_refresh_interval > 0
+    end
+
+    def personal_access_tokens_enabled?
+      config.personal_access_tokens_enabled
+    end
+
+    def restricted_access_enabled?
+      config.restricted_access_enabled
+    end
+
+    def contributors
+      @contributors ||= Octokit::Client.new(auto_paginate: true).contributors(Octokit::Repository.from_url(config.source_repo))
+    rescue
+      nil
+    end
```

Comment:
@WillAbides Note this is an actual behavior change; I thought it weird to have contributors configurable so there's no writer here, it's just a helper. I can change it back if you had some thoughts on that.